src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import cn from 'classnames';
+import { CompletedStatus } from '../types/CompletedStatus';
+
+type Props = {
+  itemsLeft: number;
+  filterByStatus: CompletedStatus;
+  onFilterByStatus: (status: CompletedStatus) => void;
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  itemsLeft,
+  filterByStatus,
+  onFilterByStatus,
+}) => (
+  <footer className="todoapp__footer" data-cy="Footer">
+    <span className="todo-count" data-cy="TodosCounter">
+      {`${itemsLeft} items left`}
+    </span>
+
+    {/* Active link should have the 'selected' class */}
+    <nav className="filter" data-cy="Filter">
+      <a
+        href="#/"
+        className={cn('filter__link', {
+          selected: filterByStatus === CompletedStatus.All,
+        })}
+        data-cy="FilterLinkAll"
+        onClick={() => onFilterByStatus(CompletedStatus.All)}
+      >
+        {CompletedStatus.All}
+      </a>
+
+      <a
+        href="#/active"
+        className={cn('filter__link', {
+          selected: filterByStatus === CompletedStatus.Active,
+        })}
+        data-cy="FilterLinkActive"
+        onClick={() => onFilterByStatus(CompletedStatus.Active)}
+      >
+        {CompletedStatus.Active}
+      </a>
```

Comment:
You can get an array of values from your enum with `Object.values` and map through it to create links